be/src/vec/functions/function_java_udf.cpp

```diff
@@ -38,7 +38,7 @@
 #include "vec/core/block.h"
 #include "vec/data_types/data_type_nullable.h"
 
-const char* EXECUTOR_CLASS = "org/apache/doris/udf/UdfExecutor";
```

Comment:
here seems error, udaf is under be/src/vec/aggregate_functions/aggregate_function_java_udaf.h

Reply:
thanks. I will fix it.